### ISSUE TITLE
New tab with inherited container is not highlighted in the sidebar

### ISSUE THREAD

**Issue by mike-tn** [piroor/treestyletab#2385]
## Short description
With some options set, after opening new tab  its background color in the sidebar is of  NOT active tab (and sometimes left edge in not highlighted also).

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. In TST options (New tab behavior) set: 
     >Other "New Blank Tab" Actions> Open new blank tab   =  "a child of the current tab"
     >Container of new blank child tabs>Inherit the container of the parent tab to a new blank child tab = "on"
 4. Open new blank tab in any container
 5. Open in it any site or leave blank
 6. Open new blank tab with Ctrl-T or with Open a new tab button in Firefox tabs bar.

## Expected result
New tab is added , it's active, it's tab in the TST sidebar is highlighted.

## Actual result
New tab is added and active, it appears both in firefox tabs bar and in TST side bar,  its highlighted in tabs bar, but in the sidebar  there are no tabs with background color of highlighted tab. Left edge of the new tab is sometimes blue but sometimes not - there are no highlighted tabs at all.
![Image 2019-09-19 15-04-20 001](https://user-images.githubusercontent.com/53944324/65242547-f6406e00-daee-11e9-8770-a74478a129a7.png)

## Environment

 * Platform (OS): Windows 7 Pro 64bit
 * Version of Firefox: 69.0.1  64bit
 * Version (or revision) of Tree Style Tab: 3.1.8

**Comment by piroor** [comment]
After a research I've understood what's happen. When a new tab 271 is opened by Ctrl-T and it is reopened as another new tab 272 with a container inherited from the active tab, got log on the view is:

~~~
treestyletab:notify-tab-activated 271
treestyletab:notify-tab-activated 272
treestyletab:notify-tab-creating 272
treestyletab:notify-tab-created 272
272 is activated
treestyletab:notify-tab-creating 271
treestyletab:notify-tab-created 271
treestyletab:notify-tab-removing 271
271 is activated
treestyletab:notify-tab-removed 271
~~~

The already closed tab 271 must not be activated at this situation...

**Comment by piroor** [comment]
This should be fixed with the commit 4cfae78.

**Comment by mike-tn** [comment]
Great thanks, seems works OK!!!
